hypha/static_src/src/javascript/apply/edit-comment.js

```diff
@@ -32,15 +35,36 @@
                 <div id="wmd-button-bar-edit-comment" class="wmd-button-bar"></div>
                 <textarea id="wmd-input-edit-comment" class="wmd-input" rows="10">${commentContents}</textarea>
                 <div id="wmd-preview-edit-comment" class="wmd-preview"></div>
+                <br>
+                <div>Visible to:</div>
+            </div>
+        `;
+
+        const radioButtonsDiv = '<div id="edit-comment-visibility"></div>';
+        let key = '';
+        let label = '';
+        let radioButtons = '';
+
+        $.each(visibilityOptions, function (idx, value) {
```

Comment:
@harumhelmy This seems to work but Javascript is not my strong suit so please take a look at it.